src/compiler/moduleNameResolver.ts

```diff
@@ -2839,6 +2839,12 @@ function loadModuleFromImmediateNodeModulesDirectory(extensions: Extensions, mod
 
 function loadModuleFromSpecificNodeModulesDirectory(extensions: Extensions, moduleName: string, nodeModulesDirectory: string, nodeModulesDirectoryExists: boolean, state: ModuleResolutionState, cache: ModuleResolutionCache | undefined, redirectedReference: ResolvedProjectReference | undefined): Resolved | undefined {
     const candidate = normalizePath(combinePaths(nodeModulesDirectory, moduleName));
+    if (moduleName.indexOf(":") > -1) {

```

Comment:
This should go at a higher level, in the `tryResolve` of `nodeModuleNameResolverWorker`...

Reply:
wont that affect export import mapping

Reply:
I’m not sure I understand. Can you give an example?

Reply:
I cant remember exactly what i meant here